website/content/docs/security/acl/acl-roles.mdx

```diff
@@ -0,0 +1,378 @@
+---
+layout: docs
+page_title: Roles
+description: >-
+  This topic describes roles within the access control list (ACL) system. A role is a named set of policies and service identities.
+  They are intended to
+---
+
+# Roles
```

Comment:
If the Node/Service Identities are moved to the policies page, and the attributes are removed, I think there ends up being very little left here. I suspect Roles are not really a "core" concept either. They are a minor concept for convenience. Maybe they would be better off as a sub-section on the Policies or Tokens page, instead of their own page.